cypress/integration/ResetPassword/index.spec.ts

```diff
@@ -0,0 +1,25 @@
+describe('Reset password page', () => {
+  describe('given the email', () => {
+    it('renders the success message', () => {
+      cy.interceptApi('/me', {
+        statusCode: 401,
+        fixture: 'requests/unauthorizedUserProfile',
+      }).as('getUserProfile');
+
+      cy.interceptApi('/passwords', {
+        fixture: 'requests/resetPassword',
+      }).as('resetPassword');
+
+      cy.visit('/password-reset');
+
+      cy.findByLabelText('Email').type('user@mail.com');
+      cy.findByRole('button', { name: /Send Recovery Email/ }).click();
+
+      cy.wait(['@getUserProfile', '@resetPassword']);
+
+      cy.findByText(
+        "We've emailed you instructions to reset your password."
+      ).should('exist');
```

Comment:
Do you have any plans to work on localization later on? just curious as this text would be appeared in the UI, unit test and UI test 😅

Reply:
For IC, I haven't planned for the localization 😅